homeassistant/components/lamarzocco/strings.json

```diff
@@ -0,0 +1,50 @@
+{
+  "config": {
+    "flow_title": "La Marzocco Espresso {host}",
+    "abort": {
+      "reauth_successful": "[%key:common::config_flow::abort::reauth_successful%]",
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
+    },
+    "error": {
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "no_machines": "No machines found in account",
+      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
+      "machine_not_found": "The configured machine was not found in your account. Did you login to the correct account?",
+      "unknown": "[%key:common::config_flow::error::unknown%]"
+    },
+    "step": {
+      "user": {
+        "data": {
+          "username": "[%key:common::config_flow::data::username%]",
+          "password": "[%key:common::config_flow::data::password%]"
+        },
+        "data_description": {
+          "password": "Your password from the La Marzocco app",
+          "username": "Your username from the La Marzocco app"
+        }
+      },
+      "machine_selection": {
+        "data": {
+          "host": "[%key:common::config_flow::data::ip%]",
+          "machine": "Machine"
+        },
+        "data_description": {
+          "host": "Local IP address of the machine"
+        }
```

Comment:
I think it would be nice if we could have a description that explains _why_ someone needs to fill in the host